{
  "myawesomelab/test/test_logger.py::test_failed_logger": true
}